docs/reference/builder.md

```diff
@@ -480,30 +480,36 @@ the first pattern, followed by one or more `!` exception patterns.
 
 ## FROM
 
-    FROM <image>
+    FROM <image> [AS <name>]
 
 Or
 
-    FROM <image>:<tag>
+    FROM <image>:<tag> [AS <name>]
```

Comment:
this was already like this, but given that tag is optional, should this be?;

   FROM <image>[:<tag>] [AS <name>]